tests/jest/unit/app/modules/kit-library/assets/js/hooks/use-kit-call-to-action-updated-url.test.js

```diff
@@ -0,0 +1,15 @@
+import useKitCallToActionUpdatedUrl from '../../../../../../../../../app/modules/kit-library/assets/js/hooks/use-kit-call-to-action-updated-url.js';
+
+test( 'useKitCallToActionUpdatedUrl should return the url with utm_term and utm_content', () => {
+	// Arrange
+	const url = 'https://elementor.com/pro/?utm_source=kit-library&utm_medium=wp-dash&utm_campaign=gopro';
```

Comment:
test URL without query params
test non-URL string
test URL where the parameters you add already exist

Reply:
Added a new test for each scenario.